gbtpipe/Gridding.py

```diff
@@ -471,6 +507,11 @@ def griddata(filelist,
     for thisfile in filelist:
         ctr += 1
         s = fits.open(thisfile)
+	# Remove outliers in Lat/Lon space
```

Comment:
Given `gbtpipe` is used in a lot of projects, this should probably be made much more optional and wrapped in a block in a `if flagSpatialOutlier:`  block.  I can see using a variety of methods eventually in this block.